hack/multikueue/create-multikueue-kubeconfig.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
```

Comment:
we already have this file under `site/static/examples/multikueue/create-multikueue-kubeconfig.sh`, either works for me:
- create a copy which is gitignored
- reference the full path
- symlink
- (another option which avoids duplication)